app/models/card.rb

```diff
@@ -4,12 +4,17 @@ class Card < ActiveRecord::Base
 
   validate :translated_properly
 
-  after_validation :days3
+  #after_validation :days3
+
+  scope :cards_older_today, -> { where("review_date < ?", Date.today)}
```

Comment:
Space missing inside }.<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.